tests/s-arg-oct.c

```diff
@@ -0,0 +1,12 @@
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+int main(void)
+{
+	const char *pathname = "bar.foo";
+	creat(pathname, 0755);
```

Comment:
We usually give an empty line after declarations.

Reply:
Add empty between lines 7 and 8 and before `return 0;` as well.